src/Backends/SystemUpdate.vala

```diff
@@ -65,6 +65,10 @@ public class SettingsDaemon.Backends.SystemUpdate : Object {
     }
 
     public async void check_for_updates (bool force, bool notify) throws DBusError, IOError {
+        if (Utils.is_running_in_demo_mode ()) {
```

Comment:
```suggestion
        if (SettingsDaemon.Utils.is_running_in_demo_mode ()) {
```

This should fix the build error

Reply:
Oops thanks, I think I fixed this locally and forgot to push :sweat_smile: